_posts/2016-08-25-hybox-names.markdown

```diff
@@ -0,0 +1,38 @@
+
+3. **turnkey repository product,** locally managed. Building on what the Hydra community has learned working on Sufia to make it easy to install and maintain, we expect to provide a complete, configurable, turnkey product--more akin to Avalon, DSpace, or even ContentDM. We need a name for this product analogous to those systems, and with broad market appeal and strong branding identity. We have put a lot of thought into requirements for a good name: 
+
+    * We want something clearly related to the Hydra project to leverage its brand, and to invite uptake and code contributions / maintenance. Further, we want adopters of the product to know it is part of the Hydra community effort, in order to provide an on-ramp for them to join / participate in the wider community effort. 
+
+    * We want something clearly distinct from the Hydra community & repository / application framework. (Just calling it "Hydra" wouldn’t work.)
+
+    * Hydra-in-a-Box and HyBox both seem poor choices here: the former is clunky and overly long. The latter is perhaps too insider-y and lacks the natural power and mass-market appeal that a name like "Avalon" brings.
+
+    * Also, Hydra-in-a-Box implies the full functionality and flexibility of the Hydra technical framework is present in the product, but all nicely wrapped up in one package. This is not true, as we’ll be making opinionated selections, based on the community’s requirements, about functionality to include. As such, we’re looking for a name that connotes one is getting a particular flavor or profile of Hydra.
+
+    * We may want to pick up on the useful trend of picking a name related to or derived from "Hydra" in some way, but distinct. Hydrangea, Hypatia, Hydraulics, Hyperspace all fall into this pattern. 
+
+    * We need to be sure to clear the obvious hurdles--no similar products with similar names, no unintentionally offensive meanings in other languages, available domain names, etc.
+
+In short, we’re looking for a product name **that has strong branding potential, will have mass market appeal, and is distinct from but is easily related to Hydra. **
+
+For now, we have started by creating a new component that augments the Hydra stack with turnkey features. The application is, fittingly (we think), named *Lerna* after the lake where the mythological Hydra lived (special thanks to Rick Johnson at Notre Dame, the first person to suggest this name).  
+
+4. **hosted service. **One of the goals of the project is to produce a "cloud-ready" version of the repository product, so that sites can make use of the platform without having to install it locally. The project partners intend to pilot a hosted service next year to kickstart this process. (The cloud-ready components and configurations will also be open source, and a good number of other sites have been in touch about potentially running this for local consortia.) We have time and space to think about this name, and will get there next year.
```

Comment:
This sentence, "so that sites can make use of the platform without having to install it locally," conflates the hosted service with running the app in the cloud. They're related, and perhaps this is too fine a distinction, but I don't want to suggest to folks that if they want to run Lerna in the cloud they need to pay for a hosted service. I might change this to something like, "so that sites can make use of the platform without having to maintain or install their own instance."